compiler/rustc_codegen_llvm/src/context.rs

```diff
@@ -328,6 +328,17 @@ pub(crate) unsafe fn create_module<'ll>(
         }
     }
 
+    // Add "kcfi-arity" module flag if KCFI arity indicator is enabled. (See
+    // https://github.com/llvm/llvm-project/pull/117121.)
+    if sess.is_sanitizer_kcfi_arity_enabled() && sess.is_sanitizer_kcfi_enabled() {
+        // KCFI arity requires LLVM 20.1.0 or later.
+        if llvm_version < (20, 1, 0) {
```

Comment:
This is not the case -- there is no tag yet in upstream LLVM that contains the feature, sadly.